src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+
+thead.addEventListener('click', (evnt) => {
+  if (evnt.target.tagName === 'TH') {
+    const headerCell = evnt.target;
+    const columnIndex = headerCell.cellIndex;
+
+    sortTable(columnIndex);
+  }
+});
+
+function toNumber(str) {
+  return +str.replace(/[$,]/g, '');
+}
+
+function sortTable(columnIndex) {
```

Comment:
Did some refactoring, this could be shortened and implicit return could be used

![image](https://github.com/mate-academy/js_sort_table_DOM/assets/87986955/47e4e4bc-1308-41eb-a0a5-11ed268637ef)

and a more functional solution with helpers piping the returned values

![image](https://github.com/mate-academy/js_sort_table_DOM/assets/87986955/3f2765bf-6684-4588-9e09-efc08b896868)